lib/SIL/Verifier/SILOwnershipVerifier.cpp

```diff
@@ -281,8 +295,11 @@ bool SILValueOwnershipChecker::gatherNonGuaranteedUsers(
                      << "Initial: " << *initialScopedOperand << "\n";
       });
     };
-    foundError |=
-        initialScopedOperand->getImplicitUses(nonLifetimeEndingUsers, &error);
+    SmallVector<Operand *, 4> borrowLifetimeEndingUsers;
+    initialScopedOperand->getImplicitUses(borrowLifetimeEndingUsers, &error);
```

Comment:
My thought: why not just make the reborrow verifier a struct that is created here that takes the initialScopedOperand and verifies based off of that? Would be simpler and details wouldn't leak out.